src/shims/dir.props

```diff
@@ -25,6 +25,6 @@
   </ItemGroup>
 
   <PropertyGroup>
-  	<GenFacadesOutputPath>$(IntermediateOutputPath)facades/</GenFacadesOutputPath>
+  	<GenFacadesOutputPath>$(BaseIntermediateOutputPath)/shims/facades/</GenFacadesOutputPath>
```

Comment:
I think we should do `'$(Configuration)'` here, also it is redundant to have shims and facades so I think I would go with `$(BaseIntermediateOutputPath)/shims/$(Configuration)`